bcda/bcdacli/cli.go

```diff
@@ -471,6 +471,17 @@ func setUpApp() *cli.App {
 				return err
 			},
 		},
+		{
+			Name:     "retrieve-suppression-hicn-to-bbid",
```

Comment:
This will need to be called after 1-800-Medicare-Import.
Instead of a new Jenkins job, I recommend running the command immediately following the ETL: https://github.com/CMSgov/bcda-ops/blob/master/jenkins_files/Jenkinsfile.import_1800medicare#L57

It will also need to be called as part of our smoke tests:
https://github.com/CMSgov/bcda-ops/blob/master/jenkins_files/Jenkinsfile.smoke_test#L273